src/Commands/UploadArchive.ts

```diff
@@ -119,8 +119,10 @@ export class UploadArchive implements CommandModule {
       // eslint-disable-next-line no-console
       console.log(await archives.upload(spec));
       process.exit(0);
-    } catch (e) {
-      logger.error(`Error during "archive:upload": ${e.message}`);
+    } catch (error) {
+      logger.error(
+        ErrorMessageBuilder.buildMessage({ command: 'archive:upload', error })
```

Comment:
The shorthand props first. Ditto elsewhere
```suggestion
        ErrorMessageBuilder.buildMessage({ error, command: 'archive:upload' })
```